feathr_project/feathr/settings.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+from jinja2 import Template
+
+
+class ObservationSettings:
+    """Time settings of the observation data. Used in feature join."""
```

Comment:
Missing pydoc. What is an `event_timestamp_column`? We need to explain this to users. Please audit and make sure documentation standards are met on all user facing Python Config APIs